var/spack/repos/builtin/packages/cgdb/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Cgdb(AutotoolsPackage):
+    """A curses front-end to GDB"""
+
+    homepage = "https://cgdb.github.io"
+    url      = "https://cgdb.me/files/cgdb-0.7.0.tar.gz"
+
+    version('0.7.0', sha256='bf7a9264668db3f9342591b08b2cc3bbb08e235ba2372877b4650b70c6fb5423')
+
+    # Required dependency
+    depends_on('ncurses',  type='build')
+    depends_on('readline', type='build')
```

Comment:
Are you sure these aren't build/link dependencies? I would be surprised if it wasn't linking to these libraries in any way.